includes/class-coblocks-site-design.php

```diff
@@ -99,8 +99,7 @@ function() {
 	 * @return boolean
 	 */
 	public static function short_circuit_check() {
-		$active_theme = wp_get_theme();
-		return 'Go' !== $active_theme->get( 'Name' );
+		return 'go' !== get_option( 'stylesheet' );
```

Comment:
We might want to use `get_stylesheet()` here instead

Reply:
@sc0ttkclark Good call